matminer/datasets/dataset_retrieval.py

```diff
@@ -8,9 +8,10 @@
              "Anubhav Jain <ajain@lbl.gov>" \
              "Daniel Dopp <dbdopp@lbl.gov>"
 
+_dataset_dict = None
 
-def load_dataset(name, data_home=None, download_if_missing=True,
-                 include_metadata=False):
+
+def load_dataset(name, data_home=None, download_if_missing=True):
```

Comment:
@ardunn Is this this `load_dataset` method in question? It does seem like a really useful one to have public.

Reply:
yes this is the one i'd like to keep public as well